sdk/ml/azure-ai-ml/azure/ai/ml/operations/_datastore_operations.py

```diff
@@ -93,9 +93,10 @@ def _list_helper(datastore_resource: Datastore, include_secrets: bool) -> Datast
         )
 
     @monitor_with_activity(ops_logger, "Datastore.ListSecrets", ActivityType.PUBLICAPI)
-    def _list_secrets(self, name: str) -> DatastoreSecrets:
+    def _list_secrets(self, name: str, expirable_secret: bool) -> DatastoreSecrets:
         return self._operation.list_secrets(
             name=name,
+            expirable_secret=expirable_secret,
```

Comment:
this would have to passed after `workspace_name` to keep python happy about positional args and such